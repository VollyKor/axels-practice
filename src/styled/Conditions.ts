import styled from 'styled-components';

export const ConditionText = styled.p`
    bottom: 0;
    position: initial;
    text-align: center;
    font-size: 0.6rem;

    @media ${({ theme }) => theme?.deviceSize?.conditionTextWidth} {
        position: absolute;
        margin-top: 0.8rem;
    } ;
`;

export const ConditionLink = styled.a`
    font-size: 0.65rem;
    color: var(--text-accent-color);
`;
