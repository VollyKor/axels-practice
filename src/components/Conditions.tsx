import { ConditionLink, ConditionText } from 'styled/Conditions';

const Conditions = () => (
    <ConditionText className="ml-2 text-muted">
        All puraches are subject to our{' '}
        <ConditionLink href="/" target="_blanc" className="text-muted">
            Terms and Conditions
        </ConditionLink>
    </ConditionText>
);

export default Conditions;
