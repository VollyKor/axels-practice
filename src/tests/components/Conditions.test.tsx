import { shallow } from 'enzyme';

import Conditions from 'components/Conditions';

it('renders coorectly', () => {
    const wrapper = shallow(<Conditions />);
    expect(wrapper).toMatchSnapshot();
});
