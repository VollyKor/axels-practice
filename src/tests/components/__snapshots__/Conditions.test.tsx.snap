// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coorectly 1`] = `
<styled.p
  className="ml-2 text-muted"
>
  All puraches are subject to our
   
  <styled.a
    className="text-muted"
    href="/"
    target="_blanc"
  >
    Terms and Conditions
  </styled.a>
</styled.p>
`;
